Assignment/solution_student/index.html

```diff
@@ -13,18 +15,85 @@
   <body>
 
     <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
+    <div class="container clearfix">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
+     <header>
+       <div class="logo">
+        <h1>Relaxr</h1>
+        <ul class="nav">
+          <li><a>About</a></li>
+          <li><a>FAQ</a></li>
+          <li><a>Team</a></li>
+          <li><a>Contact Us</a></li>
+          <li><a>Blog</a></li>
+        </ul>
+       </div> 
       </header>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div class="main">
+        <div class="column1">
+          <div class="implemented">
```

Comment:
This would be a good opportunity to use an HTML5 structural element like `<article>` since it is an article after all